neurokit2/complexity/utils.py

```diff
@@ -111,7 +111,7 @@ def _get_count_fuzzy(embedded, r, distance="chebyshev", n=1):
 # =============================================================================
 # Get R
 # =============================================================================
-def _get_r(signal, r="default", dimension=2):
+def _get_r(signal, r="default", dimension=2, show=True):
```

Comment:
For optimizing tolerance it's probably more elegant to have the `_get_r` utils function to incorporate the optimizing since this is shared across multiple entropy files - and then for other more unique parameters (such as those generated by `_fractal_correlation_get_r` and `_fractal_higuchi_optimal_k`) the optimizing can then be done in their respective complexity function files

for a visualization of the mean `r` in this case:
![Figure_1](https://user-images.githubusercontent.com/54053748/131950702-61fc4232-c755-40a4-bd76-4a79dba9b23a.png)